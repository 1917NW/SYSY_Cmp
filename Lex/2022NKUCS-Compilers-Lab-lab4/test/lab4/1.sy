int a;
/*abc*/
//
int main()
{
    int a;
    a = 0x0000aC;
    if(a < 5)
        return 1;
    return 0;
}