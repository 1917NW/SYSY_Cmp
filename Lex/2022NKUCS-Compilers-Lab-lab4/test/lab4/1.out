[DEBUG LAB4]: 	INT	int	1
[DEBUG LAB4]: 	ID	a	1	94459886441280
[DEBUG LAB4]: 	SEMICOLON	;	1
[DEBUG LAB4]: 	INT	int	3
[DEBUG LAB4]: 	ID	main	3	94459886441536
[DEBUG LAB4]: 	LPAREN	(	3
[DEBUG LAB4]: 	RPAREN	)	3
[DEBUG LAB4]: 	LBRACE	{	4
[DEBUG LAB4]: 	INT	int	5
[DEBUG LAB4]: 	ID	a	5	94459886441696
[DEBUG LAB4]: 	SEMICOLON	;	5
[DEBUG LAB4]: 	ID	a	6	94459886441696
[DEBUG LAB4]: 	ASSIGN	=	6
[DEBUG LAB4]: 	HNUM	172	6
[DEBUG LAB4]: 	SEMICOLON	;	6
[DEBUG LAB4]: 	INT	int	7
[DEBUG LAB4]: 	ID	b	7	94459886441744
[DEBUG LAB4]: 	SEMICOLON	;	7
[DEBUG LAB4]: 	LBRACE	{	8
[DEBUG LAB4]: 	INT	int	9
[DEBUG LAB4]: 	ID	b	9	94459886442000
[DEBUG LAB4]: 	SEMICOLON	;	9
[DEBUG LAB4]: 	RBRACE	}	10
[DEBUG LAB4]: 	LBRACE	{	11
[DEBUG LAB4]: 	ID	b	12	94459886441952
[DEBUG LAB4]: 	ASSIGN	=	12
[DEBUG LAB4]: 	NUM	3
[DEBUG LAB4]: 	SEMICOLON	;	12
[DEBUG LAB4]: 	RBRACE	}	13
[DEBUG LAB4]: 	IF	if	14
[DEBUG LAB4]: 	LPAREN	(	14
[DEBUG LAB4]: 	ID	a	14	94459886441904
[DEBUG LAB4]: 	LESS	<	14
[DEBUG LAB4]: 	NUM	5
[DEBUG LAB4]: 	RPAREN	)	14
[DEBUG LAB4]: 	RETURN	return	15
[DEBUG LAB4]: 	NUM	1
[DEBUG LAB4]: 	SEMICOLON	;	15
[DEBUG LAB4]: 	RETURN	return	16
[DEBUG LAB4]: 	NUM	0
[DEBUG LAB4]: 	SEMICOLON	;	16
[DEBUG LAB4]: 	RBRACE	}	17
